docs/input/index.rst

```diff
@@ -97,7 +97,7 @@ the `Mailgun API`_.
 
 .. _microsoft-input-adapter:
 
-Microsot Bot Framework input adapter
+Microsoft Bot Framework input adapter
 ====================================
```

Comment:
There needs to be one more `=` in the underline to match the length of the title.